### DISCUSSION TITLE
Arena Sport 6 CRO

### DISCUSSION THREAD

**Discussion by TurboPing** [iptv-org/community#881] [Q&A]
### Channel ID

ArenaSport6.hr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://2hubs.ddns.net:25461/BabNour/telegrramm/60948
<img width="788" alt="image" src="https://user-images.githubusercontent.com/7253922/234044460-5c16b527-a332-4203-b46d-b490aafaa922.png">

http://cloudcdn.tk:80/Visarhem/lQzhj3068/13828
<img width="787" alt="image" src="https://user-images.githubusercontent.com/7253922/234044350-2a5a3fca-498b-4d57-b7f5-aab702e0393c.png">

**Comment by kaikyoofers** [comment]
> http://2hubs.ddns.net:25461/BabNour/telegrramm/60948
> <img width="788" alt="image" src="https://user-images.githubusercontent.com/7253922/234044460-5c16b527-a332-4203-b46d-b490aafaa922.png">
> 
> http://cloudcdn.tk:80/Visarhem/lQzhj3068/13828
> <img width="787" alt="image" src="https://user-images.githubusercontent.com/7253922/234044350-2a5a3fca-498b-4d57-b7f5-aab702e0393c.png">
> 

The username (BabNour) and password (telegrramm) means its a xtream link

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19827
